orderer/common/multichannel/metrics.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package multichannel
+
+import (
+	"github.com/hyperledger/fabric/common/metrics"
+	"github.com/hyperledger/fabric/orderer/common/types"
+)
+
+var (
+	clusterStatusOpts = metrics.GaugeOpts{
+		Namespace:    "cluster",
+		Subsystem:    "",
+		Name:         "status",
```

Comment:
The name `status` is pretty hopelessly vague, unless we add more context with a subsystem or different namespace.  A cluster status of 'active' for instance does not imply that the other nodes are reachable, etc.  It is the `participation_status` or similar.

Reply:
Done. It's participation_status now with the namespace change.